src/ducks/client/links.js

```diff
@@ -8,7 +8,7 @@ import { offlineDoctypes, TRANSACTION_DOCTYPE } from 'doctypes'
 import { pickAdapter, getAdapterPlugin } from 'ducks/client/linksHelpers'
 import { getWarmupQueries } from './linksHelpers'
 
-const activatePouch = __POUCH__ && !flag('banks.pouch.disabled')
+export const getActivatePouch = () => __POUCH__ && !flag('banks.pouch.disabled')
```

Comment:
isPouchActive? Since we return a boolean... Should be added to https://github.com/cozy/cozy-guidelines#naming-of-functions